src/libs/SidebarUtils.ts

```diff
@@ -126,10 +126,9 @@ function getOrderedReportIDs(
             return;
         }
         const isSystemChat = ReportUtils.isSystemChat(report);
-        const isExpenseReportWithoutParentAccess = ReportUtils.isExpenseReportWithoutParentAccess(report);
         const shouldOverrideHidden =
             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-            hasValidDraftComment(report.reportID) || hasErrorsOtherThanFailedReceipt || isFocused || isSystemChat || report.isPinned || isExpenseReportWithoutParentAccess;
+            hasValidDraftComment(report.reportID) || hasErrorsOtherThanFailedReceipt || isFocused || isSystemChat || report.isPinned;
```

Comment:
```suggestion
            hasValidDraftComment(report.reportID) || hasErrorsOtherThanFailedReceipt || isFocused || isSystemChat || report.isPinned || ReportUtils.requiresAttentionFromCurrentUser(report, parentReportAction);
```

For this [test](https://expensify.slack.com/archives/C02NK2DQWUX/p1731591109244119?thread_ts=1731537652.304929&cid=C02NK2DQWUX), the expenses sent for approval for User B before they were removed as an approver should still appear in User B's LHN because the reports have GBRs and are awaiting User B's approval.

Reply:
Or perhaps move the early return check inside the shouldReportBeInOptionList with moving the GBR check inside the function to close to top.